<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="eapfukl325" name="sh.calaba.calabashdriver.GreenhousAndroidTests" tests="1" failures="1" timestamp="17 Jan 2013 12:25:59 GMT" time="0.001" errors="0">
  <testcase name="testSignIn" time="0.001" classname="sh.calaba.calabashdriver.GreenhousAndroidTests">
    <failure type="java.lang.AssertionError" message="The local calabash-driver server cannot be started: ">
      <![CDATA[java.lang.AssertionError: The local calabash-driver server cannot be started: 
	at org.testng.Assert.fail(Assert.java:83)
	at sh.calaba.calabashdriver.common.CalabashBaseTest$DriverServerRunnable.run(CalabashBaseTest.java:59)
	at java.lang.Thread.run(Thread.java:662)
	at sh.calaba.calabashdriver.common.CalabashBaseTest.startCalabashDriverServer(CalabashBaseTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:128)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1203)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1128)
	at org.testng.TestNG.run(TestNG.java:1036)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.net.BindException: Address already in use: bind
	at sun.nio.ch.Net.bind(Native Method)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:124)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:59)
	at org.eclipse.jetty.server.nio.SelectChannelConnector.open(SelectChannelConnector.java:196)
	at org.eclipse.jetty.server.AbstractConnector.doStart(AbstractConnector.java:362)
	at org.eclipse.jetty.server.nio.SelectChannelConnector.doStart(SelectChannelConnector.java:274)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:55)
	at org.eclipse.jetty.server.Server.doStart(Server.java:254)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:55)
	at sh.calaba.driver.server.CalabashAndroidServer.start(CalabashAndroidServer.java:125)
	at sh.calaba.calabashdriver.common.CalabashBaseTest$DriverServerRunnable.run(CalabashBaseTest.java:56)
	... 28 more
]]>
    </failure>
  </testcase> <!-- testSignIn -->
</testsuite> <!-- sh.calaba.calabashdriver.GreenhousAndroidTests -->
